app/push-sw.js

```diff
@@ -41,10 +40,30 @@ self.addEventListener('notificationclick', (evt) => {
   );
 });
 
+function recipientsToString(recipients) {
+  let result;
+  if (!recipients.length) {
+    result = 'Reminder';
```

Comment:
Is `recipients` optional? I think the UI will freak out if an empty array is passed.